zodiac-tsrp/master.toml

```diff
@@ -0,0 +1,20 @@
+[master]
+  runner = "s3://ambiata-dispensary-v2/dist/master/master-haskell/linux/x86_64/20160603003512-3d86f6c/master-haskell-20160603003512-3d86f6c"
+  version = 1
+  sha1 = "4fd979c87b2dfd7d8909ae052f3aeb66939b4bc2"
+
+[build.dist]
+  GHC_VERSION="7.10.2"
+  CABAL_VERSION = "1.22.4.0"
+
+[build.dist-7-10]
+  HADDOCK = "true"
+  HADDOCK_S3 = "$AMBIATA_HADDOCK_MASTER"
+  GHC_VERSION = "7.10.2"
+  CABAL_VERSION = "1.22.4.0"
+
+[build.branches-7-10]
+  HADDOCK = "true"
+  HADDOCK_S3 = "$AMBIATA_HADDOCK_BRANCHES"
+  GHC_VERSION = "7.10.2"
```

Comment:
Thanks - I'm not sure these should be global though, if I add more sections I don't necessarily want to use the same GHC.